components/Terminal.js

```diff
@@ -7,7 +7,8 @@ import './Terminal.css';
 console.oldLog = console['log']; // eslint-disable-line no-console, dot-notation
 
 function handleLogging(method, addToOutput) {
-  console[method] = (...args) => { // eslint-disable-line no-console
+  console[method] = (...args) => {
+    // eslint-disable-line no-console
```

Comment:
Hi @jcgertig 
I'm using prettier-eslint that on save lint the files.
However, I can change the lines of code 🙂